src/middleware.js

```diff
@@ -30,15 +31,15 @@ const send = (action: OfflineAction, dispatch, config: Config, retries = 0) => {
     .catch(error => {
       // discard
       if (config.discard(error, action, retries)) {
-        console.log('Discarding action', action.type);
+        logger.log('Discarding action', action.type);
         return dispatch(complete(metadata.rollback, false, error));
```

Comment:
How can I get a hold of this error? I get "Discarding action" in my console, but I can't see why...

Reply:
The error can be found in the rollback payload

Reply:
So action.payload.error?

Reply:
> The error object returned by the effects reconciler will be set as the payload.